src/crimson/os/seastore/seastore_types.h

```diff
@@ -217,14 +219,14 @@ constexpr paddr_t zero_paddr() {
 }
 
 struct __attribute((packed)) paddr_le_t {
-  ceph_le32 segment = ceph_le32(NULL_SEG_ID);
-  ceph_les32 offset = ceph_les32(NULL_SEG_OFF);
+  ceph_le64 segment = ceph_le64(NULL_SEG_ID);
+  ceph_les64 offset = ceph_les64(NULL_SEG_OFF);
```

Comment:
This is done following our previous discussion: [here](https://docs.google.com/document/d/1brbZ73sPBKbbsLteiuaNGRZ88G8Vp91MdnyORcb7KyM/edit)

`For now, we only consider segments as storage backends. Since we, in the future, may have to maintain multiple segments from multiple segment managers, we need to change paddr_t::segment to uint64_t and use the top N(4?) bits of paddr_t block device id. And for segment managers, the successive 28bits would be used as a segment manager id..`

Reply:
@cyx1231st I think there was a misunderstanding about this when writing the design doc of the extent placement manager, my question now is whether a 4-GB-max segment acceptable? If so, I'll drop this modification.

Reply:
I can only recall the typical segment size would be 128MB from @athanatos.

Reply:
@cyx1231st I've dropped this change, please take a look again.